### PR TITLE
PR_12

### PR DESCRIPTION
**Pull request by TestGitHub100500**
PR_12

### CODE REVIEW

**Comment by TestGitHub100500** [comment]
change

**Comment by OksanaEfanova** [comment]
comment

**Comment by OksanaEfanova** [comment]
comment7

**Comment by OksanaEfanova** [comment]
comment9

**Comment by OksanaEfanova** [comment]
comment 13

**Comment by OksanaEfanova** [comment]
comment 14

**Comment by OksanaEfanova** [comment]
comment 15

**Comment by OksanaEfanova** [comment]
comment 16

**Comment by OksanaEfanova** [comment]
comment 17

**Comment by OksanaEfanova** [comment]
comment 18

**Review by OksanaEfanova — COMMENTED** [review]
improve something

**Review by TestGitHub100500 — COMMENTED** [review]
fix

**Review by OksanaEfanova — CHANGES_REQUESTED** [review]
add1

**Review by TestGitHub100500 — COMMENTED** [review]
fix2

**Review by OksanaEfanova — APPROVED** [review]

File: PR_11 [hunk 1]
```diff
@@ -0,0 +1 @@
+PR_11
```
Comment (OksanaEfanova) [inline PR_11:1]: MOVE

**Review by OksanaEfanova — CHANGES_REQUESTED** [review]

**Review by OksanaEfanova — CHANGES_REQUESTED** [review]
change

**Comment by OksanaEfanova** [comment]
123

**Review by OksanaEfanova — CHANGES_REQUESTED** [review]
testtest

**Comment by OksanaEfanova** [comment]
dffffggfd

**Comment by OksanaEfanova** [comment]
[[[

**Comment by OksanaEfanova** [comment]
qwe

**Comment by OksanaEfanova** [comment]
test

**Comment by OksanaEfanova** [comment]
testtest

File: PR_11 (hunk 1 above)
Comment (OksanaEfanova) [reply PR_11:1]: 1

**Comment by OksanaEfanova** [comment]
3

**Comment by OksanaEfanova** [comment]
1

**Pull request closed without merging**